src/App.tsx

```diff
@@ -92,12 +82,12 @@ export const App: React.FC = () => {
               'is-8-desktop',
               'Sidebar',
               {
-                'Sidebar--open': selectedPost,
+                'Sidebar--open': post,
               },
             )}
           >
             <div className="tile is-child box is-success ">
```

Comment:
There is an extra space at the end of the className string. Consider removing it to maintain consistency.